app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -491,6 +491,7 @@
 %div{"ng-if" => "#{ng_model}.emstype == 'gce'                    || " + |
                 "#{ng_model}.emstype == 'scvmm'                  || " + |
                 "#{ng_model}.emstype == 'lenovo_ph_infra'        || " + |
+                "#{ng_model}.emstype == 'redfish_ph_infra'       || " + |
```

Comment:
I suppose this is correct place to put it?

Reply:
It is correct in the sense that the following code hides all the tabs that we do not need. I considered adding a separate block for Redfish provider, but I decided against it when I saw that other providers are already reusing the configuration block. Besides, we can always create a new block if/when we actually need to made some customizations.